src/extension.js

```diff
@@ -1,7 +1,7 @@
-/**
- * Copyright (C) Animal Logic Pty Ltd. All rights reserved.
- */
-
+/**

```

Comment:
I think vscode will attempt to convert everything to CR/LF if it finds anything with CR/LF in it - and this file was a mixture of both I think. Let me know if you have any preference...

Reply:
consistency would be nice indeed, maybe a slight preference for linux's convention?